src/stateMachine/HotFire/HotFireStateMachine.cpp

```diff
@@ -181,6 +181,22 @@ void HotFireStateMachine::ServoControlEXT() {
     END_TRANSITION_MAP
 } // clang-format on
 
+void HotFireStateMachine::setHeater(const std::shared_ptr<sensorsData> &interfaceData, bool on)
```

Comment:
Set heater function is only called on events, but needs to be called at each loop